Component/Pet/Pet.js

```diff
@@ -40,9 +52,18 @@ const StyledSpan = styled.span`
   position: absolute;
   left: 20px;
 
-  @media screen and (max-width: 1000px) {
+  @media screen and (max-width: 1024px) {
     font-size: 10vh;
   }
+
+  /* @media screen and (max-width: 768px) {
+    left: 14px;
+    font-size: 8vh;
+  } */
```

Comment:
Prior to sharing a PR, we want to remove all code that is not needed, such as commented out code:

```suggestion
```